ADR/Components/tests/test_Plane.py

```diff
@@ -0,0 +1,169 @@
+                  "linear_decay_coefficient": -0.15,
+                  "cg_x": 0.11,
+                  "cg_z": 0.01,
+                  "tpr_x": 0.13,
+                  "tpr_z": -0.08,
+                  "Iyy_TPR": 1,
+                  "CD_tp": 1,
+                  "S_tp": 1,
+                  "CD_fus": 1,
+                  "S_fus": 1,
+                  "u_k": 1}
+    plane = Plane(plane_data)
+    return plane
+
+
+def test_get_CL_alpha_plane(plane):
+    cl = {-10.0: 0.8961720610687022,
+          -9.0:  1.0828123664122136,
+          -8.0:  1.2690087022900762,
+          -7.0:  1.4544367938931297,
+          -6.0:  1.6393807633587787,
+          -5.0:  1.8236004580152672,
+          -4.0:  2.00718,
+          -3.0:  2.1898352671755723,
+          -2.0:  2.3712140458015263,
+          -1.0:  2.5516685496183205,
+          0.0:   2.7310867175572517,
+          1.0:   2.9091882442748087,
+          2.0:   3.085813282442748,
+          3.0:   3.260721679389313,
+          4.0:   3.4343535877862594,
+          5.0:   3.606312824427481,
+          6.0:   3.7766433587786263,
+          7.0:   3.945017099236641,
+          8.0:   4.111562137404579,
+          9.0:   4.276278473282442,
+          10.0:  4.438685801526717,
+          11.0:  4.59902427480916,
+          12.0:  4.75733786259542,
+          13.0:  4.836142442748091,
+          14.0:  4.913922137404579,
+          15.0:  4.99043679389313,
+          16.0:  4.977114045801526,
+          17.0:  4.9639671755725185,
+          18.0:  4.950996183206107,
+          19.0:  4.938289007633587,
+          20.0:  4.925757709923665}
+
+    df_cl_alpha = dict_to_dataframe(cl, "CL", "alpha")
+    pd.testing.assert_frame_equal(
+        plane.get_CL_alpha_plane(), df_cl_alpha, check_exact=True)
+
+
+def test_get_CD_alpha_plane(plane):
+    cd = {-10.0: 0.0059450381679389324,
+          -9.0: 0.011138015267175573,
+          -8.0: 0.017323358778625956,
+          -7.0: 0.02445328244274809,
+          -6.0: 0.03233160305343511,
+          -5.0: 0.04095832061068702,
+          -4.0: 0.050333435114503805,
+          -3.0: 0.060700916030534346,
+          -2.0: 0.07201297709923664,
+          -1.0: 0.08363328244274808,
+          0.0: 0.09644213740458016,
+          1.0: 0.1097152671755725,
+          2.0: 0.12358076335877863,
+          3.0: 0.13819465648854962,
+          4.0: 0.1535569465648855,
+          5.0: 0.16966763358778625,
+          6.0: 0.18584641221374046,
+          7.0: 0.20305770992366412,
+          8.0: 0.22033709923664122,
+          9.0: 0.23816488549618323,
+          10.0: 0.25630091603053434,
+          11.0: 0.27503312977099237,
+          12.0: 0.29387358778625955,
+          13.0: 0.30346625954198475,
+          14.0: 0.31297099236641224,
+          15.0: 0.3226279389312977,
+          16.0: 0.32042946564885494,
+          17.0: 0.31818702290076334,
+          18.0: 0.315856641221374,
+          19.0: 0.3134822900763358,
+          20.0: 0.31106396946564885}
```

Comment:
Those are the values for me:

0.005705
0.011138
0.017323
0.024453
0.032332
0.040958
0.050333
0.060701
0.072013
0.083633
0.096442
0.109715
0.123581
0.138635
0.153757
0.169668
0.185846
0.203058
0.220337
0.238165
0.256301
0.275033
0.293874
0.303226
0.312971
0.322388
0.320189
0.317947
0.315616
0.313242
0.310824